moon/moon-view.sh

```diff
@@ -1 +1 @@
-python3 -m osgar.tools.lidarview --pose2d app.pose2d --lidar rosmsg.scan --camera rosmsg./scout_1/camera/left/image_raw/compressed --camera2 rosmsg./scout_1/camera/right/image_raw/compressed --lidar-limit 20000 --window-size 1200 660 --title rosmsg.rot $1  
```

Comment:
well, this change is a bit temporary, in particular if we decide to use renamed messages like `left_image`

Reply:
Absolutely, or at least some of the channels that change should be parameters so that I don't have to edit the script every time I switch from hauler to excavator.  I got tired of stashing this every time I wanted to switch a branch and there are no scouts at this phase of development.